doc/contributing/security-release-process.md

```diff
@@ -43,191 +43,118 @@ The current security stewards are documented in the main Node.js
-  ```
-
-* [ ] Post in the [nodejs-social channel][]
-  in the OpenJS slack asking for amplification of the blog post.
-  ```text
-  Security release:
-
-  New security releases are now available for versions <add versions> of Node.js.
-
-  https://nodejs.org/en/blog/vulnerability/month-year-security-releases/
-  ```
-
-* [ ] Comment in [docker-node][] issue that release is ready for integration.
-  The docker-node team will build and release docker image updates.
-
-* [ ] For every H1 report resolved:
-  * Close as Resolved
-  * Request Disclosure
-  * Request publication of [H1 CVE requests][]
-    * (Check that the "Version Fixed" field in the CVE is correct, and provide
-      links to the release blogs in the "Public Reference" section)
-  * In case the reporter doesn't accept the disclosure follow this process:
-    * Remove the original report reference within the reference text box and
-      insert the public URL you would like to be attached to this CVE.
-    * Then uncheck the Public Disclosure on HackerOne box at the bottom of the
-      page.
-      ![screenshot of HackerOne CVE form](https://github.com/nodejs/node/assets/26234614/e22e4f33-7948-4dd2-952e-2f9166f5568d)
-
-* [ ] PR machine-readable JSON descriptions of the vulnerabilities to the
-  [core](https://github.com/nodejs/security-wg/tree/HEAD/vuln/core)
-  vulnerability DB. _**LINK TO PR**_
-  * For each vulnerability add a `#.json` file, one can copy an existing
-    [json](https://github.com/nodejs/security-wg/blob/0d82062d917cb9ddab88f910559469b2b13812bf/vuln/core/78.json)
-    file, and increment the latest created file number and use that as the name
-    of the new file to be added. For example, `79.json`.
-
-* [ ] Close this issue
-
-* [ ] Make sure the PRs for the vulnerabilities are closed.
-
-* [ ] PR in that you stewarded the release in
-  [Security release stewards](https://github.com/nodejs/node/blob/HEAD/doc/contributing/security-release-process.md#security-release-stewards).
-  If necessary add the next rotation of the steward rotation.
+1. **Lock down the CI:**
+   * Lock down the CI to prevent public access to the CI machines, ping a member of `@nodejs/jenkins-admins`.
+
+2. **Release:**
+   * Verify the CI is green on all release proposals (test-V8, CITGM, etc).
+   * Follow the [release process](https://github.com/nodejs/node/blob/main/doc/contributing/releases.md).
+
+3. **Unlock the CI:**
+   * Unlock the CI to allow public access to the CI machines, ping a member of `@nodejs/jenkins-admins`.
+
+4. **Publish Post-Release Blog Post:**
+   * Publish the post-release blog post on the `nodejs/nodejs.org` repository.
+
+5. **Notify the community:**
+   * Notify the community about the upcoming security release:
+     * Slack: `#nodejs-social`
+     * [docker-node](https://github.com/nodejs/docker-node/issues)
+     * [build-wg](https://github.com/nodejs/build/issues)
+     * Email: notify <oss-security@lists.openwall.com>
```

Comment:
Is this still manual, if so it would be good to say that and to say to comment on the issues that were automatically  created in the earlier step

Also is `nodejs-sec` missing from the list?